src/datasources/transaction-api/transaction-api.service.spec.ts

```diff
@@ -173,41 +173,4 @@ describe('TransactionApi', () => {
       expect(httpErrorFactory.from).toHaveBeenCalledTimes(1);
     });
   });
-
```

Comment:
Why are these tests removed? They still relevant, as the endpoint still exist (but it uses NetworkService directly instead of CacheFirstDatasource), or am I missing something?

Reply:
You're correct. I've added them again in the context of not caching in 8469ed0a094b62c49b0ca23355456cc35c32e3ad.